src/main.h

```diff
@@ -237,7 +237,7 @@ void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, bool fRejectAbsurdFee=false);
+                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, CFeeRate maxFeeRate=CFeeRate(0));
```

Comment:
Actually, this is where I dislike the default value the most, I got confused with CWallet's method declaration. In that method I don't even care if fRejectAbsurdFee=true is preserved.